lib/viget/deployment/recipes/slack_notification.rb

```diff
@@ -0,0 +1,38 @@
+require 'viget/deployment/slack_notifier'
+
+Capistrano::Configuration.instance.load do
+  set :slack_url, 'https://hooks.slack.com/services/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'
```

Comment:
Thoughts on putting this in [`core.rb`](https://github.com/vigetlabs/viget-deployment/blob/master/lib/viget/deployment/recipes/core.rb) so it's not hidden here?